common/src/main/java/com/wynntils/core/features/overlays/BasicOverlay.java

```diff
@@ -12,29 +12,39 @@
 
 public class BasicOverlay extends Overlay {
     private final QuadConsumer<Overlay, PoseStack, Float, Window> renderConsumer;
+    private final QuadConsumer<Overlay, PoseStack, Float, Window> renderPreviewConsumer;
 
     public BasicOverlay(
```

Comment:
Do we need both the float/float and the OverlaySize constructors? If anything, I'd rather see a simplified constructor without the renderPreviewConsumer argument (which uses the renderConsumer for both). But if we create that variant, and we want it in both constructors, we're up to four constructors which seems a bit too much...